CONTRIBUTING.md

```diff
@@ -53,6 +53,10 @@ For now `main` is the main branch and code improvements are made in topic branch
 
 A maintainer will review your code and merge it when it has the required number of approvals.
 
+## Hot Reloading
+
+We make use of the [Inject](https://github.com/krzysztofzablocki/Inject) framework for hot reloading debug builds. To set up hot reloading, follow the [documentation](https://github.com/krzysztofzablocki/Inject?tab=readme-ov-file#individual-developer-setup-once-per-machine).
```

Comment:
Maybe this documentation is too light, but I am kinda rushing to throw this PR out there. Maybe the reviewer can tell me if it's enough or document the steps they had to take. 

For instance, the first thing their documentation says to do is add some linker flags which is already done. So at least that part is superfluous. It's been a while since I set this up so I don't remember what I did exactly.